packages/babel-preset-env/test/fixtures/preset-options/hermes-0_7/input.mjs

```diff
@@ -0,0 +1,21 @@
+class Foo {
+  #bar() {
+    // Should be transpiled
+  }
+}
+
+try {
+  /** Should be transpiled */
+  function scoped() {}
+
+  /** Should be transpiled */
+  let arrow = () => scoped;
+} /** Should be transpiled */ catch {}
```

Comment:
Is there a way to only transpile what is really required?

I don't know why this should be transpiled, because try-catch-without-catch-args is supported?

![image](https://github.com/babel/babel/assets/5236548/f7f517d1-eca0-48c5-9f2b-fa9b13faa4a9)

Reply:
Hermes supports optional catch binding(what you mean by `try-catch-without-catch-args`) in 
hermes v0.12.0. See compat-table change: https://github.com/compat-table/compat-table/pull/1832.

So the `preset-env` will transform `try {} catch {}` when targeting hermes 0.7 but will not when targeting hermes 0.12.